### ISSUE TITLE
论文复现 使用文档-单机多卡训练问题

### ISSUE THREAD

**Issue by caopulan** [PaddlePaddle/Paddle#38099]
https://www.paddlepaddle.org.cn/documentation/docs/zh/guides/02_paddle2.0_develop/06_device_cn.html#danjiduokaxunlian

单机多卡基础API部分代码有问题
`train_loader = paddle.io.DataLoader(train_dataset, batch_size=32, shuffle=True)`
应改用 paddle.io.DistributedBatchSampler

**Comment by littletomatodonkey** [comment]
感谢反馈，我们修复下
